packages/components/vite.config.ts

```diff
@@ -31,7 +31,9 @@ export default defineConfig({
         environment: 'happy-dom',
         setupFiles: ['./src/setupTests.ts'],
         css: true,
+        include: ['./src/**/*.test.ts', './src/**/*.test.tsx'],
         coverage: {
+            exclude: [...configDefaults.exclude, '**.**', 'scripts/templates/**/*'],
```

Comment:
Why `**.**`? Is it possible to remove it?

Reply:
that way it doesn't check the files in the root directory for coverage, such as tailwind config, postcss config etc.

Reply:
I modified the selector to more specific